packages/data-context/src/sources/GitDataSource.ts

```diff
@@ -89,31 +88,46 @@ export class GitDataSource {
     debug('config: %o', this.config)
     try {
       this.#git = simpleGit({ baseDir: this.config.projectRoot })
-      this.#gitBaseDir = this.config.projectRoot
     } catch {
-      this.#git = simpleGit()
+      // suppress exception if git cannot be found
```

Comment:
I hit another spot in the code where an exception was suppressed and it took me foreverrrrrsssss to track it down. Is there any way we can surface an error, i.e. the error still shows in the console, but it does not impact the UI?

Reply:
`debug` added